samples/dnn/facial_landmarks_ie.cpp

```diff
@@ -0,0 +1,195 @@
+    if (argc == 1 || parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    //Parsing input arguments
+    const std::string facexmlPath = parser.get<std::string>("facestruct");
+    const std::string facebinPath = parser.get<std::string>("faceweights");
+    const Target faceTarget = parser.get<Target>("facetarget");
+    const Backend faceBackend = parser.get<Backend>("facebackend");
+
+    const std::string landmxmlPath = parser.get<std::string>("landmstruct");
+    const std::string landmbinPath = parser.get<std::string>("landmweights");
+    const Target landmTarget = parser.get<Target>("landmtarget");
+    const Backend landmBackend = parser.get<Backend>("landmbackend");
+
+    const Scalar color = Scalar(parser.get<int>("b"), parser.get<int>("g"), parser.get<int>("r"));
+
+    //Models' definition & initialization
+    Net faceNet = readNet(facexmlPath, facebinPath);
+    faceNet.setPreferableTarget(faceTarget);
+    faceNet.setPreferableBackend(faceBackend);
+    const unsigned int faceObjectSize = 7;
+    const float faceConfThreshold = 0.7f;
+    const unsigned int facecols = 672;
+    const unsigned int facerows = 384;
+
+    Net landmNet = readNet(landmxmlPath, landmbinPath);
+    landmNet.setPreferableTarget(landmTarget);
+    landmNet.setPreferableBackend(landmBackend);
+    const unsigned int landmcols = 60;
+    const unsigned int landmrows = 60;
+    const float bb_enlarge_coefficient = 1.0f;
+
+    const std::string win1 = "FaceLandmarkDetector";
+    //TODO: rewrite color
+    const Scalar black(0,0,0);
+
+    //Input
+    VideoCapture cap;
+    if (parser.has("input"))
+        cap.open(parser.get<String>("input"));
+    else if (!cap.open(0))
+            {
+                std::cout<<"No input available"<<std::endl;
+                return 1;
+            }
+
+    Mat img;
+    while (waitKey(1) < 0)
+    {
+        cap >> img;
+        if (img.empty())
+        {
+           waitKey();
+           break;
+        }
+
+        Mat shwimg;         //Image to show
+        img.copyTo(shwimg);
+
+        //Preprocessing
+        Mat inputImg;
+        resize(img, inputImg, Size(facecols, facerows));
+
+        //Infering Facedetector
+        faceNet.setInput(blobFromImage(inputImg));
+        Mat outFaces = faceNet.forward();
+
+        float* faceData = (float*)(outFaces.data);
+        for (size_t i = 0; i < outFaces.total(); i += faceObjectSize)
+        {
+            //Face rectangle prediction processing
+            float confidence = faceData[i + 2];
+            if (confidence > faceConfThreshold)
+            {
+                int left = int(faceData[i + 3] * img.cols);
+                left = std::max(left, 0);
+                int top = int(faceData[i + 4] * img.rows);
+                top = std::max(top, 0);
+                int right  = int(faceData[i + 5] * img.cols);
+                right = std::min(right, img.cols-2);
```

Comment:
Please stick to the one convention about using or not the spaces between operators and operands